aries-site/src/layouts/main/Layout.js

```diff
@@ -251,6 +251,15 @@ export const Layout = ({
                   about this page.`,
                 }}
               />
+              <Question
+                name="email"
+                kind="textInput"
+                label="Email"
+                formProps={{
+                  help: `Feel free to leave contact
```

Comment:
going to come back and visit if we need to have a `help` or the label email is good enough